packages/editor-ui/src/components/Node.vue

```diff
@@ -778,6 +826,45 @@ export default defineComponent({
 		}
 	}
 
+	&.touch-active,
+	&:hover,
+	&.menu-open {
+		.node-options {
+			opacity: 1;
+		}
+	}
+
+	.node-options {
+		:deep(.button) {
+			--button-font-color: var(--color-text-light);
+			--button-border-radius: 0;
+		}
+		cursor: default;
+		position: absolute;
+		top: -34px;
```

Comment:
To avoid using arbitrary number here you could do `bottom: 100%` instead to have it rendered right above the node